jumpscale/packages/vdc_dashboard/frontend/components/vdcSettings/Wallet.vue

```diff
@@ -142,6 +142,9 @@ module.exports = {
         });
     },
   },
+  created() {
+    this.getQRCode();
```

Comment:
In my opinion, this call shouldn't be in the created hook.

AFAIK, in created hook, the props are created but didn't get their value yet. so the `this.wallet.address` will be `undefine` that will raise an error.

Reply:
Thanks for repoting will double check it and fix the issue

Reply:
Fixed, Please review change and resolve issue